.travis.yml

```diff
@@ -1,17 +1,35 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "pypy"
-script: make test
+script: tox
 before_install:
   - openssl version
+  # Install Google Cloud Platform SDK to enable App Engine tests
+  - if [[ $TOXENV == 'gae' && ! -d $HOME/gcloud/google-cloud-sdk ]]; then
+      mkdir -p $HOME/gcloud &&
+      wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud &&
```

Comment:
Our older releases tend to roll off the server rather quickly (this is actually currently breaking tests in my own project because I tried to pin a specific version). Even if you did, the ``gcloud components update`` command right after would update to the latest SDK anyway. In general, though, the GAE SDK pathologically avoids breaking APIs (with the exception of the dev_appserver2 switchover).